src/components/Common/Summary.tsx

```diff
@@ -0,0 +1,26 @@
+import { cls } from '@/utils';
+import { PropsWithChildren, ReactNode } from 'react';
+
+interface OrderSummaryProps {
+  title: string;
+  totalPrice: string;
+  transparent?: boolean;
+  renderButton?: (buttonText?: string) => ReactNode;
```

Comment:
```suggestion
  rightAddon?: ReactNode;
```
(마이너) 버튼에 국한되지 않고 인터페이스를 작성해보면 어떨까요?
프로젝트에서 인터페이스를 통일해두면 코드를 작성할 때 덜 혼란스럽더라구요 :)